jena-text/pom.xml

```
@@ -122,6 +133,7 @@
           <includes>
             <include>**/TS_*.java</include>
           </includes>
+          <argLine>${jena.text.test.argline}</argLine>
```

```suggestion
          <argLine>@{argline} ${jena.text.test.argline}</argLine>
```
because mockito also needs set the `argLine` for surefire. See #2772

`@{argLine}` must be defined else it blows up.

(jena-text does not use mockito at the moment - this is guessing the future)

This whole javaagent/hardening java thing is going to be bumpy!